chore(kb, grid):kb extentions for grid columns rendering (#331)
telerik/blazor-docs: knowledge-base/grid-column-in-custom-component-is-last.md

All parent components are initialized before their child components.

> On initialization, child components can't be rendered before the parent components. This is why the order in the actual code can't affect the order on initialization - the grid receives the `GridColumn` instances in the order in which their components are initialized. The columns that are added later in the component lifecycle will also be last.

Reviewer: Should be "order **of** initialization":

```suggestion
On initialization, child components can't be rendered before the parent components. This is why the order in the actual code can't affect the order of initialization - the grid receives the `GridColumn` instances in the order in which their components are initialized. The columns that are added later in the component lifecycle will also be last.
```

Also, the following statement may raise the question "What determines the order of column initialization, if it isn't the order in the code?":

"the actual code can't affect the order of initialization - the grid receives the `GridColumn` instances in the order in which their components are initialized"

Finally, what does "added later in the component lifecycle` mean? Is it programmatic addition of columns or just "later" in the code?

> On initialization, child components can't be rendered before the parent components. The grid receives the `GridColumn` instances in the order in which their components are initialized. Components rendering determines the order of the columns initialization. Columns that are added programmatically later in the component lifecycle will also be last.

Reviewer: ```suggestion
On initialization, child components are always rendered after their parent component. On the other hand, the grid receives the `GridColumn` instances in the order in which their components are initialized. This determines why columns in a custom component will be initialized and rendered last. Furthermore, columns that are added programmatically later in the component lifecycle will also be rendered last.
```

}
````

> >tip If you want to have conditional columns, you can use the [`Visible`]({%slug grid-columns-visible%}) parameter. With a certain condition, for example, a ternary operator or a method that returns a bool.

Reviewer: The second sentence follows Bulgarian grammar rules (no subject, no action), but that approach is not used in English.

```suggestion
>tip If you want to have conditional columns, you can use the [`Visible`]({%slug grid-columns-visible%}) parameter. The condition can be a ternary operator or a method that returns a bool.
```
